frontend/src/metabase/admin/datamodel/components/GuiQueryEditor/GuiQueryEditor.jsx

```diff
@@ -225,9 +225,12 @@ export class GuiQueryEditor extends Component {
 
     return (
       <div
-        className={
-          "GuiBuilder-section GuiBuilder-data flex align-center arrow-right"
-        }
+        className={cx(
+          CS.arrowRight,
+          CS.alignCenter,
+          CS.flex,
+          "GuiBuilder-section GuiBuilder-data",
```

Comment:
2 things here.
1. Please try to preserve the CSS order, so it's easier to compare to previous versions.
2. Please also coordinate this with @deniskaber since his PR migrated these 2 classes

Reply:
Yep, of course. I think it will be ok in terms of migration since I'm only making these small changes, and preserving the order of the classes _should_ help make sure there isn't a merge conflict here